src/main/java/core/basesyntax/servise/interfaces/FileService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.servise;
```

Comment:
typo error, rename the package
```suggestion
package core.basesyntax.service;
```